sql/core/src/test/resources/sql-tests/inputs/number-format.sql

```diff
@@ -0,0 +1,13 @@
+-- Verifies how we parse numbers
+
+-- parse as ints
+select 1, -1;
```

Comment:
What if the query need to read data from some pre-defined tables? do you have a plan for this feature?